src/lando/main/models/commit_map.py

```diff
@@ -0,0 +1,90 @@
+import requests
+from django.db import models
+
+from lando.main.models import BaseModel
+from lando.main.models.repo import Repo
+
+
+class CommitMap(BaseModel):
+    """Map a git hash to an hg hash, based on a specific repo."""
+
+    HGMO_PUSHLOG_TEMPLATE = "https://hg.mozilla.org/{}/json-pushes"
+    REPO_MAPPING = (("firefox", "mozilla-unified"),)
```

Comment:
Could we put this in the `settings`? (The `REPO_MAPPING` will be tricky... Maybe some mapping such as `REPO_MAPPING_firefox=mozilla-unified` that we parse and put back into an array of tuples.

Reply:
Worse comes to worse, it's one deployment away (: